Storage/BaseRepository.cs

```diff
@@ -53,6 +65,36 @@ public Task<IEnumerable<T>> GetAllAsync<T>(string tableName)
             }
         }
 
+        public Task<IEnumerable<T>> GetSomeAsync<T>(
```

Comment:
And can you add integration test for this method?

Unit tests should be independent with each other, so I suggest create something and then call this method to see if it can query the correct data back

Reply:
Will add in a separate PR.